tasks/test-project/test-project

```diff
@@ -188,74 +180,40 @@ const globalTasks = () =>
           }
         },
       },
-      {
-        title: '[link] Building Redwood framework',
-        task: async () => {
-          try {
-            await execa(
-              'yarn build:clean && yarn build',
-              [],
-              getExecaOptions(RW_FRAMEWORKPATH)
-            )
-          } catch (e) {
-            console.log('Failed to build framework...')
-            console.log()
-            console.log(
-              'Please check your branch is building with yarn build:clean && yarn build'
-            )
-            throw new Error('Failed to build framework')
-          }
-        },
-        enabled: () => link,
-      },
-      {
-        title: '[link] Adding framework dependencies to project',
-        task: () =>
-          addFrameworkDepsToProject(RW_FRAMEWORKPATH, OUTPUT_PROJECT_PATH),
-        enabled: () => link,
-      },
       {
         title: 'Installing node_modules',
         task: async () => {
           return execa('yarn install', getExecaOptions(OUTPUT_PROJECT_PATH))
         },
       },
       {
-        title: '[link] Copying framework packages to project',
-        task: () =>
-          copyFrameworkPackages(RW_FRAMEWORKPATH, OUTPUT_PROJECT_PATH),
-        enabled: () => link,
+        title: 'Tarsync the framework to the project',
+        task: () => {
+          return execa(
+            'yarn rwfw project:tarsync',
+            [],
+            getExecaOptions(OUTPUT_PROJECT_PATH),
+          )
```

Comment:
I think we still need the `enabled: () => link` no?

Reply:
Yeah thanks for spotting!